components/infobox/commons/infobox_person.lua

```diff
@@ -238,7 +238,6 @@ function Person:createInfobox()
 			)))
 
 	if self:shouldStoreData(args) then
-		self:_definePageVariables(args)
```

Comment:
was it checked that changing process order (lpdb before wiki var) doesn't do any harm?
i.e. was it checked that there is no reading from wiki var reading in `adjustLPDB` nor `getStorageType` in the customs?